bridgetown-core/test/source/custom-cache-dir/Bridgetown/Cache/Bridgetown--Cache/b7/9606fb3afea5bd1609ed40b622142f1c98125abcfe89a76a661b0e8e343910

```diff
@@ -0,0 +1,2 @@
+I"�
+{"root_dir"=>"/home/alexanderadam/projects/0-public_hosted/github.com/bridgetown/bridgetown-core/test/source", "plugins_dir"=>"/home/alexanderadam/projects/0-public_hosted/github.com/bridgetown/bridgetown-core/test/source/plugins", "source"=>"/home/alexanderadam/projects/0-public_hosted/github.com/bridgetown/bridgetown-core/test/source/src", "destination"=>"/home/alexanderadam/projects/0-public_hosted/github.com/bridgetown/bridgetown-core/test/dest", "collections_dir"=>"", "cache_dir"=>"../../custom-cache-dir", "layouts_dir"=>"_layouts", "data_dir"=>"_data", "components_dir"=>"_components", "partials_dir"=>"_partials", "collections"=>{"posts"=>{"output"=>true, "sort_direction"=>"descending", "permalink"=>"pretty"}, "pages"=>{"output"=>true, "permalink"=>"/:locale/:path/"}, "data"=>{"output"=>false}}, "taxonomies"=>{"category"=>{"key"=>"categories", "title"=>"Category"}, "tag"=>{"key"=>"tags", "title"=>"Tag"}}, "autoload_paths"=>["/home/alexanderadam/projects/0-public_hosted/github.com/bridgetown/bridgetown-core/test/source/src/_components", "/home/alexanderadam/projects/0-public_hosted/github.com/bridgetown/bridgetown-core/test/plugin_content/components"], "eager_load_paths"=>[], "autoloader_collapsed_paths"=>[], "plugins_use_zeitwerk"=>false, "include"=>[".htaccess", "_redirects", ".well-known"], "exclude"=>[".sass-cache", ".bridgetown-cache", "gemfiles", "Gemfile", "Gemfile.lock", "node_modules", "vendor/bundle/", "vendor/cache/", "vendor/gems/", "vendor/ruby/"], "keep_files"=>[".git", ".svn", "_bridgetown"], "encoding"=>"utf-8", "markdown_ext"=>"markdown,mkdown,mkdn,mkd,md", "strict_front_matter"=>false, "slugify_mode"=>"pretty", "future"=>false, "unpublished"=>false, "ruby_in_front_matter"=>true, "content_engine"=>"resource", "markdown"=>"kramdown", "highlighter"=>"rouge", "port"=>"4000", "host"=>"127.0.0.1", "base_path"=>"/", "show_dir_listing"=>false, "available_locales"=>[:en], "default_locale"=>:en, "prefix_default_locale"=>false, "permalink"=>"pretty", "timezone"=>nil, "quiet"=>false, "verbose"=>false, "defaults"=>[], "liquid"=>{"error_mode"=>"warn", "strict_filters"=>false, "strict_variables"=>false}, "kramdown"=>{"auto_ids"=>true, "toc_levels"=>[1, 2, 3, 4, 5, 6], "entity_output"=>"as_char", "smart_quotes"=>"lsquo,rsquo,ldquo,rdquo", "input"=>"GFM", "hard_wrap"=>false, "guess_lang"=>true, "footnote_nr"=>1, "show_warnings"=>false, "include_extraction_tags"=>false, "mark_highlighting"=>true}, "components_load_paths"=>["/home/alexanderadam/projects/0-public_hosted/github.com/bridgetown/bridgetown-core/test/plugin_content/components", "/home/alexanderadam/projects/0-public_hosted/github.com/bridgetown/bridgetown-core/test/source/src/_components"]}:ET
```

Comment:
I don't think you meant to check this file in
```suggestion
```

Reply:
You're of right of course :grimacing: